### ISSUE TITLE
Finish disclose row in ContentTypeUsageView

### ISSUE THREAD

**Issue by Fifok** [fortedigital/Optimizely.ContentUsage#38]
Work on after #36 
Disclose row doesn't display action buttons. I propose to create new issue for this.

_Originally posted by @Fifok in https://github.com/fortedigital/EpiContentUsage/pull/36#discussion_r1124076790_
